api/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto

```diff
@@ -216,6 +220,13 @@ message RedisProxy {
     repeated string commands = 4;
   }
 
+  // Configuration to limit reconnection rate to redis server to protect redis server
+  // from client reconnection storm.
+  message ConnectionRateLimit {
+    // Reconnection rate per sec. Rate limiting is implemented with TokenBucket.
+    uint32 connection_rate_limit_per_sec = 1;
```

Comment:
nit: I wonder whether it would be more user-friendly to add a boolean flag for enabled so that the user doesn't need to infer the enabled state from the rate limit setting.

Reply:
As per the previous API reviewer, it would be more concise to check if "ConnectionRateLimit" is set instead of adding an additional "enabled" field, which seems to be the case for other config fields here as well

Reply:
Makes sense, thanks!